R/util.R

```diff
@@ -947,8 +947,11 @@ validate_data <- function(types, data){
   TRUE
 }
 
-# used in model building functions to create meta data
-# the meta data is used for column type validation
+# This is used in model building functions
+# to create meta data.
+# It contains terms to create model and
+# column types of the variables.
+# Its's used for column type validation.
```

Comment:
Example of data structure would help understanding what this is doing.
Based on our discussion today, comment like following would help.

# return value looks like following example.
# list(type = list(col1 = "numeric", col2 = "character"))

I think this is important enough to be explained well since it becomes basis of how we deal with model.